src/app/components/Uploader/index.styles.tsx

```diff
@@ -0,0 +1,53 @@
+import { css, Theme } from '@emotion/react';
+import pixelsToRem from '#app/utilities/pixelsToRem';
+
+const styles = {
+  container: ({ spacings, mq }: Theme) =>
+    css({
+      margin: `${spacings.TRIPLE}rem ${spacings.FULL}rem`,
+      [mq.GROUP_2_MIN_WIDTH]: {
+        marginLeft: `${spacings.DOUBLE}rem`,
+        marginRight: `${spacings.DOUBLE}rem`,
+      },
+      [mq.GROUP_4_MIN_WIDTH]: { marginLeft: 0, marginRight: 0 },
+    }),
+  card: ({ spacings, palette }: Theme) =>
+    css({
```

Comment:
Please could we add a trasparent border so we retain the 'card' affordance when colours are changed in Firefox etc? As per https://5d28eb3fe163f6002046d6fa-pbeguainwl.chromatic.com/iframe.html?args=&id=components-optimopromos-topstoriessections--list-top-stories&viewMode=story

Reply:
Yep done! I have used the mq.HIGH_CONTRAST media query